File: Python/iniciante/2164fibonaccirapido.py
import math as m
n = int(input())
p1 = ((1 + m.sqrt(5)) / 2) ** n
p2 = ((1 - m.sqrt(5)) / 2) ** n
fib = (p1 - p2) / m.sqrt(5)
print("%.1lf" % fib)

File: C/Iniciante/2164fibonaccirapido.c
#include <stdio.h>
#include <math.h>
int main(){
    int n;
    double p1,p2,fib;
    scanf("%d",&n);
    p1=pow(((1+sqrt(5.0))/2),n);
    p2=pow(((1-sqrt(5.0))/2),n);
    fib=(p1-p2)/sqrt(5.0);
    printf("%.1lf",fib);
    return 0;
}